_guides/guides.md

```diff
@@ -39,8 +39,8 @@ permalink: /guides/
 
 ### Cloud
 
-- [Deploying Knative Application to Kubernetes or OpenShift]({{site.baseurl}}/guides/getting-started-knative-guide)
-- [Deploying Applications on Kubernetes]({{site.baseurl}}/guides/kubernetes-guide)
+- [Deploying native Applications on Kubernetes or OpenShift]({{site.baseurl}}/guides/kubernetes-guide)
+- [Deploying native Applications on Knative Kubernetes or OpenShift]({{site.baseurl}}/guides/getting-started-knative-guide)
```

Comment:
Apparently, they went for the "let's capitalize everything except on/or/with..." convention so could you capitalize `Native`?

Otherwise, it looks good, I'll merge it.

Reply:
done